students/mia/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model): 
+    name          = models.CharField(max_length=50)
+    email         = models.EmailField(max_length =100, unique=True)
+    password      = models.CharField(max_length=100)
+    mobile_number = models.CharField(max_length=50)
+    created_at    = models.DateTimeField(auto_now_add=True)
+    updated_at    = models.DateTimeField(auto_now_add=True)
```

Comment:
```suggestion
    updated_at    = models.DateTimeField(auto_now=True)
```

최초 생성 일자(created_at)가 아닌 수정 일자(updated_at) 에는 속성을 `auto_now=True`로 사용합니다!

예은님께서 작성하신 모델링 코드를 본다면 created_at, updated_at 두 컬럼 모두 똑같은 기능을 합니다.